index.html

```diff
@@ -5502,7 +5504,100 @@ <h2>Base Context</h2>
+identify the type of the <code>verifiableCredential</code> in which it
+appears, i.e., to indicate which set of claims the
+<code>verifiableCredential</code> contains. This property, and the value <code>VerifiableCredential</code> within the set of its values, are mandatory. Whilst
+it is good practice to include one additional value depicting the unique
+subtype of this verifiableCredential, it is permitted to either omit
+or include additional type values in the array.
+Since many verifiers will request verifiableCredentials of a specific
+subtype, then omitting the subtype value might make it more difficult for
+verifiers to inform the holder which verifiableCredential they require.
+When a verifiableCredential is of multiple subtypes, then listing all of
+them in the type property is sensible. However, the usage of the
+<code>type</code> property in a JSON-LD representation of a credential
+conveys more than what it means in a JSON representation of a credential.
+With JSON-LD, the technology is not only describing the categorization of the
+set of claims, the technology is also conveying the structure of the sub-graph
+of the claims.
+        </p>
+
+        <p>
+The primary purpose of the <code>@context</code> property, from a JSON-LD
+perspective, is to convey the meaning of the data and term definitions of
+the data in a verifiable credential in a machine readable way. When
+utilizing only a JSON representation the <code>@context</code> provides
+little technical benefits, but remains mandatory. The <code>@context</code>
+property is used to map the globally unique URIs for
+<code>verifiableCredential</code> and <code>verifiablePresentation</code>
+properties into short form alias names making both the JSON and JSON-LD
+representations of them more human-friendly to read. From a JSON-LD
+perspective, this mapping also allows for the data in a
+<code>verifiableCredential</code> to be modelled in a network of
+machine-readable data by enhancing how the data in the
+<code>verifiableCredential</code> or <code>verifiablePresentation</code>
+relates to a larger, machine-readable data graph, which is useful for being
+able to tell machines how to relate the meaning of data to other
+data in an ecosystem where parties are unable to coordinate. This
+property, with the first value in the set being
+<code>https://www.w3.org/2018/credentials/v1</code>, is mandatory.
+        </p>
+
+        <p>
+Since the <code>@context</code> property is used to map data to a graph
+data model and the <code>type</code> property in JSON-LD is used to
+describe nodes within the graph, the <code>type</code> property becomes
+even more important when using the two in combination. For example, if the
+<code>type</code> property is not included within the resolved contexts
+using JSON-LD, it could lead to claims being dropped and no longer being
+integrity protected during the production and consumption of the
```

Comment:
Alternatively...
```suggestion
using JSON-LD, it could lead to claims being dropped and/or their integrity
no longer being protected during the production and consumption of the
```

Reply:
added the text into the PR now - I was having trouble directly committing the suggestion using Github's normal process.